code/__DEFINES/vendors.dm

```diff
@@ -1,30 +1,26 @@
-#define MARINE_CAN_BUY_UNIFORM (1<<0)
-#define MARINE_CAN_BUY_SHOES (1<<1)
-#define MARINE_CAN_BUY_HELMET (1<<2)
-#define MARINE_CAN_BUY_ARMOR (1<<3)
-#define MARINE_CAN_BUY_GLOVES (1<<4)
-#define MARINE_CAN_BUY_EAR (1<<5)
-#define MARINE_CAN_BUY_BACKPACK (1<<6)
-#define MARINE_CAN_BUY_R_POUCH (1<<7)
-#define MARINE_CAN_BUY_L_POUCH (1<<8)
-#define MARINE_CAN_BUY_BELT (1<<9)
-#define MARINE_CAN_BUY_GLASSES (1<<10)
-#define MARINE_CAN_BUY_MASK (1<<11)
-#define MARINE_CAN_BUY_ESSENTIALS (1<<12)
-#define MARINE_CAN_BUY_SECONDARY (1<<13)
-#define MARINE_CAN_BUY_ATTACHMENT (1<<14)
-#define MARINE_CAN_BUY_MRE (1<<15)
-#define MARINE_CAN_BUY_ACCESSORY (1<<16)
+#define MARINE_CAN_BUY_UNIFORM "uniform"
+#define MARINE_CAN_BUY_SHOES "shoes"
+#define MARINE_CAN_BUY_HELMET "helmet"
+#define MARINE_CAN_BUY_ARMOR "armor"
+#define MARINE_CAN_BUY_GLOVES "gloves"
+#define MARINE_CAN_BUY_EAR "ear"
+#define MARINE_CAN_BUY_BACKPACK "backpack"
+#define MARINE_CAN_BUY_POUCH "pouch"
+#define MARINE_CAN_BUY_BELT "belt"
+#define MARINE_CAN_BUY_GLASSES "glasses"
+#define MARINE_CAN_BUY_MASK "mask"
+#define MARINE_CAN_BUY_ESSENTIALS "essentials"
+#define MARINE_CAN_BUY_SECONDARY "secondary"
+#define MARINE_CAN_BUY_ATTACHMENT "attachment"
+#define MARINE_CAN_BUY_MRE "mre"
+#define MARINE_CAN_BUY_ACCESSORY "accessory"
+#define MARINE_CAN_BUY_COMBAT_SHOES "combat_shoes"
+#define MARINE_CAN_BUY_COMBAT_HELMET "combat_helmet"
+#define MARINE_CAN_BUY_COMBAT_ARMOR "combat_armor"
+#define MARINE_CAN_BUY_KIT "kit"
+#define MARINE_CAN_BUY_DRESS "dress"
 
-#define MARINE_CAN_BUY_COMBAT_SHOES (1<<17)
-#define MARINE_CAN_BUY_COMBAT_HELMET (1<<18)
-#define MARINE_CAN_BUY_COMBAT_ARMOR (1<<19)
-#define MARINE_CAN_BUY_COMBAT_R_POUCH (1<<20)
-#define MARINE_CAN_BUY_COMBAT_L_POUCH (1<<21)
-
-#define MARINE_CAN_BUY_KIT (1<<22)
-
-#define MARINE_CAN_BUY_ALL ((1<<23) - 1)
+#define MARINE_CAN_BUY_ALL list(MARINE_CAN_BUY_UNIFORM = 1, MARINE_CAN_BUY_SHOES = 1, MARINE_CAN_BUY_HELMET = 1, MARINE_CAN_BUY_ARMOR = 1, MARINE_CAN_BUY_GLOVES = 1, MARINE_CAN_BUY_EAR = 1, MARINE_CAN_BUY_BACKPACK = 1, MARINE_CAN_BUY_POUCH = 2, MARINE_CAN_BUY_BELT = 1, MARINE_CAN_BUY_GLASSES = 1, MARINE_CAN_BUY_MASK = 1, MARINE_CAN_BUY_ESSENTIALS = 1, MARINE_CAN_BUY_SECONDARY = 1, MARINE_CAN_BUY_ATTACHMENT = 1, MARINE_CAN_BUY_MRE = 1, MARINE_CAN_BUY_ACCESSORY = 1, MARINE_CAN_BUY_COMBAT_SHOES = 1, MARINE_CAN_BUY_COMBAT_HELMET = 1, MARINE_CAN_BUY_COMBAT_ARMOR = 1, MARINE_CAN_BUY_KIT = 1, MARINE_CAN_BUY_DRESS = 99)
```

Comment:
Formatting of the list would be good

Reply:
Can try to do it but it's just gonna be super long vertically instead of horizontally.. And i try to avoid messing with whitespaces in defines :\